ui/src/flux/containers/CheckServices.tsx

```diff
@@ -75,31 +87,27 @@ export class CheckServices extends PureComponent<Props & WithRouterProps> {
     return sources.find(s => s.id === params.sourceID)
   }
 
-  private overlay() {
-    const {showOverlay, services} = this.props
+  private get renderOverlay(): JSX.Element {
+    const {showOverlay} = this.state
 
-    if (services.length) {
-      return
-    }
-
-    showOverlay(
-      <OverlayContext.Consumer>
-        {({onDismissOverlay}) => (
-          <FluxOverlay
-            mode="new"
-            source={this.source}
-            onDismiss={onDismissOverlay}
-          />
-        )}
-      </OverlayContext.Consumer>,
-      {}
+    return (
```

Comment:
it looks like it's no longer checking to see if there are services; should this be the case?